components/SearchBox/index.js

```diff
@@ -1,22 +1,20 @@
 'use strict';
 
 var React = require('react');
-var bem = require('../BemHelper')('algolia-magic-search-box');
+var bem = require('../BemHelper')('as-search-box');
 var cx = require('classnames');
 
-var AlgoliasearchHelper = require('algoliasearch-helper/src/algoliasearch.helper');
-var SearchResults = require('algoliasearch-helper/src/SearchResults');
-
 var SearchBox = React.createClass({
   propTypes: {
-    helper: React.PropTypes.instanceOf(AlgoliasearchHelper),
-    results: React.PropTypes.instanceOf(SearchResults),
-    onFocus: React.PropTypes.func,
```

Comment:
Sorry I removed onFocus without any specific commit. This was done to clean the widget API before we decide we do need an onFocus prop and expose it in the widget.